### PR TITLE
Develop draw shapes

### PR DESCRIPTION
**Pull request by Sjoerddbb**
Added all features for drawing shapes on VGA screen

### CODE REVIEW

File: CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c [hunk 1]
```diff
@@ -6,27 +6,221 @@
...
+/**
+ * @fn int DrawBitmap(uint8_t, uint8_t, uint8_t)
+ * @brief Draws a bitmap on given location
+ *
+ * @param xp
+ * @param yp
+ * @param selection
+ * @return
+ */
+int DrawBitmap(uint8_t selector,uint8_t xp, uint8_t yp)
```
Comment (ruubruben) [inline CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c]: check style sheet

Comment (ruubruben) [reply CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c]: if (foo == bar){
}

File: CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c [hunk 2]
```diff
@@ -6,27 +6,221 @@
  *      Author: Sjoerd
  */
 #include <main.h>
+#include <math.h>
+#include <Bitmaps.h>
```
Comment (yopocho) [inline CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c]: files cannot have capital letters, please use lower case only

File: CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c [hunk 3]
```diff
@@ -6,27 +6,221 @@
...
- * @param yp1
+ * @param xp
+ * @param yp
+ * @param Width
+ * @param Heigth
+ * @param color
+ * @param filled
+ * @return
+ */
+int DrawRectangle(uint8_t xp, uint8_t yp, uint8_t Width, uint8_t Heigth, uint8_t color, uint8_t filled)
```
Comment (ruubruben) [inline CubeIDE/Swont_ide_uart_6.5/swont_ide/Core/ub_lib/DrawShapes.c]: void no int

**Review by ruubruben — CHANGES_REQUESTED** [review]
Stylesheet for functions and statements

**Review by yopocho — APPROVED** [review]
LGTM

**Review by ruubruben — APPROVED** [review]
Viet er goed uit Sjoerd

**Pull request merged**